src/common/resources/voucher.js

```diff
@@ -20,3 +21,21 @@ export const VOUCHER_LINES = {
   path: VOUCHER_LINES_API,
   records: 'voucherLines',
 };
+
+export const VOUCHER_NUMBER_START = {
+  ...BASE_RESOURCE,
+  path: 'voucher-storage/voucher-number/start',
+  POST: {
+    headers: {
```

Comment:
what the reason of these headers?

Reply:
Because this request doesn't work with other headers. By default we use 'application/json' but not for this

Reply:
it should work. It makes no sense to add ad-hoc fix to this very place. Please contact with back-end guys to file a ticket, it looks like a back-end issue.